modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
+	err := e.db.View(func(tx *bolt.Tx) error {
+		var height types.BlockHeight
+		err := encoding.Unmarshal(tx.Bucket(bucketInternal).Get(internalBlockHeight), &height)
+		if err != nil {
+			return err
+		}
+		block, exists := e.cs.BlockAtHeight(height)
```

Comment:
based on the state of the database, I don't think there's a way to avoid this. Given that we frequently load the most recent block facts for a block (it's the homepage), we really shouldn't have to go to disk to figure out the id of the most recent block, it should just be kept in memory as the consensus set updates.